v2/tools/generator/internal/astmodel/type_visitor_test.go

```diff
@@ -236,6 +245,94 @@ func TestMakeTypeVisitorWithInjectedFunctions(t *testing.T) {
 			OptionalIntType,
 			"optional",
 		},
+		{
+			"EnumTypeHandler",
+			NewEnumType(StringType),
+			func(builder *TypeVisitorBuilder) {
+				builder.VisitEnumType = func(tv *TypeVisitor, ot *EnumType, _ interface{}) (Type, error) {
```

Comment:
Parameter naming should be based on the provided `Type` - `ot` means `ObjectType` or `OneOfType`.

```suggestion
				builder.VisitEnumType = func(tv *TypeVisitor, et *EnumType, _ interface{}) (Type, error) {
```